docs/docs-ref-autogen/outlook/office.appointmentread.yml

```diff
@@ -560,6 +511,67 @@ items:
       <tr><td>[Applicable Outlook
       mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)</td><td>Appointment Attendee</td></tr>
       </table>
+
+      #### Examples
+
+
+      ```javascript
+
+      // The following code passes a string to the `displayReplyForm` function.
+
+      Office.context.mailbox.item.displayReplyForm('hello there');
+
+      Office.context.mailbox.item.displayReplyForm('<b>hello there</b>');
+
+
+      // Reply with an empty body.
+
+      Office.context.mailbox.item.displayReplyForm({});
+
+
+      // Reply with just a body.
+
+      Office.context.mailbox.item.displayReplyForm(
+
+      {
+          'htmlBody' : 'hi'
+      });
+
+
+      // Reply with a body and a file attachment.
+
+      Office.context.mailbox.item.displayReplyForm(
+
+      {
+          'htmlBody' : 'hi',
+          'attachments' :
+          [
+              {
+                  'type' : Office.MailboxEnums.AttachmentType.File,
+                  'name' : 'squirrel.png',
+                  'url' : 'http://i.imgur.com/sRgTlGR.jpg'
+              }
+          ]
+      });
+
+
+      // Reply with a body and an item attachment.
+
+      Office.context.mailbox.item.displayReplyForm(
```

Comment:
Same extra blank lines comment.

Reply:
@AlexJerabek: It just looks spread out in the .yml file but on the site it looks like this:
![image](https://user-images.githubusercontent.com/28541963/52890711-1d8b2300-313b-11e9-8d1f-290494622ed9.png)